www/litebrowser/files/litehtml.patch

```diff
@@ -1,49 +1,64 @@
 Don't use bundled litehtml. Use MacPorts litehtml.
---- CMakeLists.txt.orig	2021-11-16 15:00:48.000000000 -0600
-+++ CMakeLists.txt	2022-06-03 13:53:45.000000000 -0500
-@@ -15,8 +15,8 @@
+diff --git CMakeLists.txt CMakeLists.txt
+index bffe7ef..43f3581 100644
+--- CMakeLists.txt
++++ CMakeLists.txt
+@@ -16,8 +16,8 @@ set(CMAKE_CXX_FLAGS_RELEASE "-O3")
  set(CMAKE_C_FLAGS_RELEASE "-O3")
  
  set(LITEBROWSER_PATH src)
 -set(LITEHTML_PATH litehtml)
--set(CONTAINER_PATH ${LITEHTML_PATH}/containers/linux)
+-set(CONTAINER_PATH ${LITEHTML_PATH}/containers/cairo)
 +set(LITEHTML_PATH ${CMAKE_INSTALL_PREFIX})
-+set(CONTAINER_PATH ${LITEHTML_PATH}/include/litehtml/containers/linux)
++set(CONTAINER_PATH ${LITEHTML_PATH}/include/litehtml/containers/cairo)
  
- pkg_check_modules(LB_LIBS REQUIRED fontconfig gdkmm-3.0 gtkmm-3.0 libcurl cairo pango pangocairo)
+ pkg_check_modules(LB_LIBS REQUIRED gdkmm-3.0 gtkmm-3.0 libcurl cairo pango pangocairo)
```

Comment:
If it no longer checks for fontconfig then I guess it should be removed from `depends_lib` too.

Reply:
comment not addressed

Reply:
Yeah, I missed these back then, and now will address everything, hence changed status to draft.

Reply:
Done.